Sources/ScipioKit/Producer/PIF/PIFCompiler.swift

```diff
@@ -115,7 +116,7 @@ struct PIFCompiler: Compiler {
             toolchain: toolchain,
             destinationTriple: toolchain.triple,
             flags: .init(),
-            enableParseableModuleInterfaces: true,
+            enableParseableModuleInterfaces: buildOptions.enableLibraryEvolution,
```

Comment:
Why do we need to pass `buildOptions.enableLibraryEvolution` to `enableParseableModuleInterfaces`? I could not find any relation of them

Reply:
Sorry, I explained the context here. https://github.com/giginet/Scipio/pull/47/files#r1106665406

`enableParseableModuleInterfaces` indicates whether enabling library-evolution or not internally.
https://github.com/apple/swift-package-manager/blob/2dc6407d38bc8d2a864baec035029a29fb36d146/Sources/Build/BuildDescription/SwiftTargetBuildDescription.swift#L466

However, this value doesn't effect our product.